internal/cli/kraft/cloud/compose/build/build.go

```diff
@@ -138,6 +138,9 @@ func Build(ctx context.Context, opts *BuildOptions, args ...string) error {
 				pkgName,
 			)
 		}
+		if !strings.HasSuffix(pkgName, ":latest") {
+			pkgName = strings.TrimSuffix(pkgName, ":latest")
+		}
```

Comment:
so you remove it if it's not set?

Reply:
Whoops

Reply:
I've now dropped this commit. The subsequent commit fixes this issue implicitly.